### DISCUSSION TITLE
LFC - LF - IACC index

### DISCUSSION THREAD

**Discussion by JessicaFerrari89** [Ifsttar/I-Simpa#320] [Q&A]
Hi, 
about tutorial 2: I find LFC and LF index clicking on advanced sound level on receiver, but I need to "divided" the value also for each sources. So I need LFC and LF index for R1S1, R1S2, R2S1, R2S2 ecc.. (where R is the receiver and S the source).
How can I do it?

And I need the same values for IACC index? Where I can found them?

Thank you so much for your support.

**Comment by Picaut** [comment]
For calculating the LFC/LF for each source, you can disable the other sources and run a new calculation.
The IACC is not calculated in I-Simpa.

**Comment by JessicaFerrari89** [comment]
Great! Thank you!
